.eslintrc.json

```diff
@@ -0,0 +1,36 @@
+{
+    "root": true,
+    "parser": "babel-eslint",
+    "extends": ["airbnb-base", "plugin:prettier/recommended"],
+    "env": {
+      "node": true,
+      "es6": true,
+      "mocha": true
+    },
+    "plugins":["prettier"],
```

Comment:
is there a specific reason for adding this plugin?

Reply:
This plugin will help in reducing linting errors and increasing the amount of error that can be fixed automatically with the --fix option.